contrib/actions/generate-target-filters.py

```diff
@@ -31,6 +31,12 @@
         f"targets/{target}"
     ] + common
 
+    if target.startswith('x86-'):
+        _filter[target].append('targets/x86.inc')
+
+    if target.startswith('bcm27xx-'):
+        _filter[target].append('targets/bcm27xx.inc')
```

Comment:
Hmm... on the one hand I'd like to have a more generic solution like the following (untested):
```py
if '-' in target:
    _filter[target].append('targets/{}.inc'.format(target.split('-')[0]))
```

On the other hand, we are building more targets than necessary now: `x86-geode` does not include `x86.inc`.